drivers/include/vcnl40x0.h

```diff
@@ -0,0 +1,147 @@
+﻿/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_vcnl40x0 VCNL40X0
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the VCNL40X0 sensor
```

Comment:
could you please add a list of supported devices: does this driver work with 4010, 4020 and 4040?